### PR TITLE
feat: Support ormb cmd proxy

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/registry/client/client.go [hunk 1]
```diff
@@ -16,14 +16,16 @@ limitations under the License.
 package client
 
 import (
-	"github.com/spf13/viper"
-	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/kubernetes"
 
-	clientset "github.com/caicloud/temp-model-registry/pkg/clientset/clientset/versioned"
+	modeljobv1alpha1 "github.com/caicloud/temp-model-registry/pkg/clientset/clientset/versioned"
+	seldonv1 "github.com/seldonio/seldon-core/operator/client/machinelearning.seldon.io/v1/clientset/versioned"
```
Comment (gaocegege) [inline pkg/registry/client/client.go:22]: Please move seldon to the upper group

Comment (simon-cj) [reply pkg/registry/client/client.go]: PTAL

**Pull request merged**